app/views/users/index.html.haml

```diff
@@ -10,10 +11,13 @@
 #result
   %h3= text(".head")
   %p= text(".instruct")
+
   - if @users.any?
-    = render "search/results_table", order_for: true, users: @users
+    = render "search/results_table",
+      order_for: current_facility.try(:single_facility?),
```

Comment:
Ah: Billing Administrators can `:manage` `User` cross-facility. I'll put in an exception for `switch_to`. What about `create`?